src/rust/bitbox02-rust/src/hww/api/show_mnemonic.rs

```diff
@@ -25,7 +26,7 @@ use bitbox02::keystore;
 
 const NUM_RANDOM_WORDS: u8 = 5;
 
-fn create_random_unique_words(word: &str, length: u8) -> (u8, Vec<&str>) {
+fn create_random_unique_words(word: &str, length: u8) -> (u8, Vec<String>) {
```

Comment:
Spent like 5min now re-reading the body of this function to remember what the `word` is and what it all does. Would've appreaciated some doc comments a lot.

Reply:
:+1: Added docs.